docs/admin_guide.md

```diff
@@ -237,6 +240,8 @@ realpath_cache_ttl = 600
 Optionally also enable OPCache for improved performances with PHP:
 
 ```ini
+zend_extension=opcache
```

Comment:
I may be missing something but isn't this already added/loaded when installing `php8.2-opcache` package? (from quick check in php sury that the guide recommends this should've been done already), or was there some situation where this isn't the case?

Reply:
On my system it wasn't enabled yet; I'm pretty sure I followed the instructions as written and when I got to this stage that line was still commented out. I can try a quick fresh install and investigate more though (e.g. maybe there's an uncommented version of the line in some other .ini file that's installed by php8.2-opcache or something) (or maybe it's a Debian vs Ubuntu difference).

Reply:
Yep - it is already enabled. Not in /etc/php/8.2/fpm/php.ini (where it is commented out) but in /etc/php/8.2/mods-available/opcache.ini. So yes, my uncommenting of that line is not needed as it was already on by installing the package, you're right.

Reply:
Please remove this line again in that case. if you use Nginx `opcache.enable=1` in PHP-FPM php.ini file should be sufficient .

Reply:
Done 👍🏻